src/components/Stats/StatsComponent.tsx

```diff
@@ -163,8 +163,10 @@ export class StatsComponent extends React.Component<WidgetProps> {
                 } else if (type === CARTA.StatsType.SumSq) {
                     unitString = `(${unit})^2`;
                 } else if (type === CARTA.StatsType.FluxDensity) {
-                    if (unit === "Jy/beam") {
-                        unitString = "Jy";
+                    if (unit.toLowerCase().includes("beam") && !unit.toLowerCase().includes("km/s")) {
```

Comment:
might be more elegant to use regex here?

Reply:
I change to the regex way to find the matching string.

Reply:
I was thinking of doing it in one line with regex and groups. If you use separate `match` calls and `!!`, then it's simpler to just use `includes`.

Reply:
Should I move back to the previous method: using unit.toLowerCase().include("beam")?

Reply:
Are we just removing a case-insensitive `/beam` here?

You can replace everything by `unitString = unit.replace(/\/beam/i, "");`